docs/python/src/class/class.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import random
+
+from PySide2.QtGui import QGuiApplication
+from PySide2.QtQml import QQmlApplicationEngine, qmlRegisterType
+from PySide2.QtCore import QUrl
+
+from PySide2.QtCore import QObject, Signal, Slot
+
+class NumberGenerator(QObject):
+    def __init__(self):
+        QObject.__init__(self)
+    
+    nextNumber = Signal(int)
```

Comment:
To respect both API (Python and Qt), we try to leave the camelCase just for the Qt API, and when local variables are used, we use_underscores. The same apply for methods/functions, etc.
I would change this property for `next_number` and also the following method for `give_number`.

Reply:
Would you consider the signal and method part of the Python API? They are intended for QML, and not Python. In the properties example, I do what you propose, e.g. setMaxNumber vs set_max_number.

Reply:
Signal and Slot are special Qt abstractions, that's why we defined with the capital first letter, here we assume QML as Qt, so QML things should keep the camelCase from Qt.

Reply:
Then I'll keep the nextNumber and giveNumber as camel case, as they are intended for QML.

Reply:
ah! I thought you asked me about the method `Signal()` not the name of the variable. I found good to have declared signals with camelCase since most of the Qt-signals using two words do the same, I guess so far it's OK, we don't have a strong feeling regarding that at the moment.